### PR TITLE
Add corsikaio recipe

### PR DESCRIPTION
**Pull request by maxnoe**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by maxnoe** [comment]
@conda-forge-admin, please ping team

- python `@conda-forge/help-python`

**Comment by maxnoe** [comment]
The build fails because it is looking for a license file in `about/license`. But I configured `license_file: LICENSE` which is where it is in the tarball.

Any idea what is going wrong?

File: recipes/corsikaio/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (ocefpaf) [inline recipes/corsikaio/meta.yaml:1]: Please remove all the comments from the example.

File: recipes/corsikaio/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  description: |
+    Library for reading the CORSIKA binary output files into meaningful
+    python data structures using numpy.
+  doc_url: https://github.com/fact-project/pycorsikaio
+  dev_url: https://github.com/fact-project/pycorsikaio
+
+extra:
+  recipe-maintainers:
+    - maxnoe
+    - kosack
```
Comment (ocefpaf) [inline recipes/corsikaio/meta.yaml:52]: @kosack do you agree to maintain this recipe?

File: recipes/corsikaio/meta.yaml (hunk 1 above)
Comment (maxnoe) [reply recipes/corsikaio/meta.yaml]: done

File: recipes/corsikaio/meta.yaml (hunk 2 above)
Comment (kosack) [reply recipes/corsikaio/meta.yaml:47]: yes I agree

**Pull request merged**